test/e2e/config/vsphere.yaml

```diff
@@ -34,29 +34,29 @@ providers:
   - name: cluster-api
     type: CoreProvider
     versions:
-      - name: v1.7.0
+      - name: v1.7.0 # latest published non-pre-release in the v1beta1 series
         # Use manifest from source files
-        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.7.0-rc.0/core-components.yaml"
+        value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.7.0/core-components.yaml"
```

Comment:
We still want to use `{go://sigs.k8s.io/cluster-api@latest-v1.7}` here I think. (also for the other v1.7.0 ones)

Reply:
@chrischdi Have to change back to hardcoded format as [here](https://github.com/kubernetes-sigs/cluster-api/pull/9265/files#diff-3ed7f0b5549fcb0a0881b8d4ff4d7cd843cc4506fb014fc460019645b0cba01bR270)